chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  a["networks"]["ironic"] = ta["networks"]["ironic"]
```

Comment:
Can you add a check that the network doesn't already exists? As people can define their own network, it could and we don't want to override their data.